Classes/Etg24/EventSourcing/Auditing/CommandLogger.php

```diff
@@ -0,0 +1,74 @@
+<?php
+namespace Etg24\EventSourcing\Auditing;
+
+use Etg24\EventSourcing\Command\Command;
+use Etg24\EventSourcing\Command\Handler\CommandHandler;
+use TYPO3\Flow\Annotations as Flow;
+
+/**
+ * Command Auditing
+ */
+class CommandLogger {
+
+	/**
+	 * @Flow\Inject
```

Comment:
Another thing, for consistency sake, @var must go before annotations. I know this differs from the TYPO3 CGL, but that's just how it is.